task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +95,8 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if(a + b <= c || a + c <= b || b + c <= a) return false;
```

Comment:
It is necessary to observe indents after if.
Also you can return expression result without return true/false